model-optimizer/extensions/ops/mvn.py

```diff
@@ -29,55 +30,58 @@ class MVN(Op):
     def __init__(self, graph: Graph, attrs: dict):
         super().__init__(graph, {
             'kind': 'op',
-            'type': __class__.op,
-            'op': __class__.op,
-            'version': 'opset2',
+            'type': self.op,
+            'op': self.op,
+            'version': 'opset6',
             'eps': None,
-            'across_channels': None,
-            'normalize_variance': 1,
-            'axes': None,
-            'in_ports_count': 1,
+            'normalize_variance': None,
+            'eps_mode': None,
+            'in_ports_count': 2,
             'out_ports_count': 1,
-            'infer': __class__.infer
+            'infer': self.infer
         }, attrs)
 
     def supported_attrs(self):
-        return ['eps', 'across_channels', 'normalize_variance', 'axes']
+        return ['eps', 'eps_mode', 'normalize_variance']
 
     def backend_attrs(self):
-        return ['eps',
-                ('across_channels', lambda node: bool_to_str(node, 'across_channels')),
-                ('normalize_variance', lambda node: bool_to_str(node, 'normalize_variance'))]
+        version = self.get_opset()
+        if version == 'opset2':
+            return ['eps',
+                    ('across_channels', lambda node: bool_to_str(node, 'across_channels')),
+                    ('normalize_variance', lambda node: bool_to_str(node, 'normalize_variance'))]
+        elif version == 'opset6':
+            return ['eps', 'eps_mode', ('normalize_variance', lambda node: bool_to_str(node, 'normalize_variance'))]
+        else:
+            raise Error('Unsupported MVN opset version "{}"'.format(version))
 
     @staticmethod
     def infer(node: None):
-        input_shape = node.in_node(0).shape
         name = node.soft_get('name', node.id)
 
-        if node.axes is not None and node.across_channels is not None:
-            raise Error('Either axes or across_channels can be set for the MVN in node "{}".'.format(name))
+        assert node.eps is not None, 'MVN required attribute `eps` unspecified for node {}'.format(name)
+        assert node.eps_mode is not None, 'MVN required attribute `eps_mode` unspecified for node {}'.format(name)
+        assert node.normalize_variance is not None, \
+            'MVN required attribute `normalize_variance` unspecified for node {}'.format(name)
+
+        PermuteInputs().set_input_permutation(node.in_node(1), node, 'input:0', 'axis')
+        copy_shape_infer(node)
```

Comment:
Also, I propose to implement value inference that is not difficult using numpy primitives and add tests for it.

Reply:
I would consider this is a minor comment because I can hardly imagine the situation when the MVN gets all constant inputs.